File: polyglot/poker_deck.cs
using System;
using System.Collections.Generic;

public class poker_deck
{
  static readonly string[] Suits = { "♠", "♥", "♦", "♣" };
  static readonly string[] Ranks = { "A", "K", "Q", "J", "10", "9", "8", "7", "6", "5", "4", "3", "2" };

  public record Card(string suit, string rank);

  static List<Card> BuildDeck()
  {
    var deck = new List<Card>(52);
    foreach (var s in Suits)
      foreach (var r in Ranks)
        deck.Add(new Card(s, r));
    return deck;
  }

  static Card Draw(List<Card> deck, Random rnd)
  {
    var idx = rnd.Next(deck.Count);
    var c = deck[idx];
    deck.RemoveAt(idx);
    return c;
  }

  public static void Main()
  {
    var rnd = new Random();
    var deck = BuildDeck();
    var c1 = Draw(deck, rnd);
    var c2 = Draw(deck, rnd);

    // Tiny JSON output (no deps)
    Console.Write("{\"hand\":[");
    Console.Write($"{{\"suit\":\"{c1.suit}\",\"rank\":\"{c1.rank}\"}},");
    Console.Write($"{{\"suit\":\"{c2.suit}\",\"rank\":\"{c2.rank}\"}}");
    Console.Write($"],\"remaining\":{deck.Count}}}");
  }
}

File: polyglot/poker_deck.java
import java.util.ArrayList;
import java.util.List;
import java.util.Random;

public class poker_deck {
  static final String[] SUITS = new String[] {"♠", "♥", "♦", "♣"};
  static final String[] RANKS = new String[] {"A", "K", "Q", "J", "10", "9", "8", "7", "6", "5", "4", "3", "2"};

  static class Card {
    final String suit;
    final String rank;
    Card(String suit, String rank) { this.suit = suit; this.rank = rank; }
  }

  static List<Card> buildDeck() {
    List<Card> deck = new ArrayList<>(52);
    for (String s : SUITS) for (String r : RANKS) deck.add(new Card(s, r));
    return deck;
  }

  static Card draw(List<Card> deck, Random rnd) {
    int idx = rnd.nextInt(deck.size());
    return deck.remove(idx);
  }

  public static void main(String[] args) {
    Random rnd = new Random();
    List<Card> deck = buildDeck();
    Card c1 = draw(deck, rnd);
    Card c2 = draw(deck, rnd);

    // Tiny JSON output (no dependencies)
    System.out.print("{\"hand\":[");
    System.out.print("{\"suit\":\"" + c1.suit + "\",\"rank\":\"" + c1.rank + "\"},");
    System.out.print("{\"suit\":\"" + c2.suit + "\",\"rank\":\"" + c2.rank + "\"}");
    System.out.print("],\"remaining\":" + deck.size() + "}");
  }
}

